.travis.yml

```diff
@@ -1,15 +1,24 @@
-sudo: true
-dist: trusty
+# Build matrix / environment variable are explained on:
+# http://about.travis-ci.org/docs/user/build-configuration/
+# This file can be validated on:
+# http://lint.travis-ci.org/
+sudo: required
+
 language: ruby
+
 rvm:
   - 2.3
 
 services:
   - docker
 
+env:
+  - TERRAFORM_VERSION=0.11.1 IMAGE_NAME=azure-loadbalancer-module
```

Comment:
I'd remove this section all together (15-16).

Reply:
I've removed the unused TERRAFORM_VERSION variable, but I think keeping the IMAGE_NAME variable is valid since it is used more than once.

Reply:
We need the TERRAFORM_VERSION variable so the developer can control what terraform run-time is contained in the image.

Reply:
When you use the dockerhub image to do the testing, you can specify what version of terraform you want to test with via the Travis ENV var.